src/app/client/src/app/modules/learn/services/assessment/assessment-score.service.ts

```diff
@@ -79,11 +79,9 @@ export class AssessmentScoreService {
    */
   private checkContentForAssessment() {
     if (_.get(this._batchDetails, 'batchId') && _.get(this._contentDetails, 'identifier') && _.get(this._batchDetails, 'courseId')) {
-      if (this._contentDetails && _.get(this._contentDetails, 'contentType') === 'SelfAssess') {
-        this.initialized = true;
-      } else {
-        this.initialized = false;
-      }
+      this.initialized = false;
```

Comment:
why do we need an if and else condition if we do that same in both the case?

Reply:
It was wrong check, I have made changes.